deliverytiersuite/delivery-tier-suite/metadata/src/main/java/webapp/WEB-INF/beans.xml

```diff
@@ -155,4 +155,13 @@
       autowire="constructor">
    </bean>
 
+	<!-- Using Liquibase to create database tables -->
```

Comment:
@rgarg24  It seems like this is just commented out?  Also are your main change log files missing from this pr?

Reply:
I commented out this because currently it is not in use. But now I am removing the commented out portion. Yes main files got missing because they were new files and they were in unversioned files. But now I am pushing them.

Reply:
In order for this fix to work it can't be commented out though right?